src/components/generic/Button/Button.jsx

```diff
@@ -0,0 +1,47 @@
+import React, {JSX} from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * Button component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {Function} props.action  OnClick callback
+ * @param {JSX.Element} props.children  List of children
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {boolean} props.isDisabled  Flag indicating button disabled
+ * @param {string} props.title Tooltip to be displayed
+ * @param {'default'|'join'|'cancel'|'ghost'} [props.type='default']  Button type
+ * @returns {object}  JSX of the element
+ */
+export default function Button({
+  action,
+  children,
+  className,
+  isDisabled,
+  title,
+  type,
+}) {
+  const cssClasses = webexComponentClasses(`-button button--${type}`, className);
```

Comment:
there's no need for a dash before 'button'

Reply:
also, `button--${type}` can be added as an 'otherBaseClass' (see the webexComponentClasses function in helpers.js)